cmd/inspect.go

```diff
@@ -57,10 +57,10 @@ func init() {
 
 	var inspectCommand = &cobra.Command{
 		Use:   "inspect <path> [<path> [...]]",
-		Short: "Inspect OPA bundle(s)",
-		Long: `Inspect OPA bundle(s).
+		Short: "Inspect OPA bundle(s) or Rego files.",
+		Long: `Inspect OPA bundle(s) or Rego files.
 
-The 'inspect' command provides a summary of the contents in OPA bundle(s). Bundles are
+The 'inspect' command provides a summary of the contents in OPA bundle(s) or a single Rego file. Bundles are
```

Comment:
I'm wondering what if we extend this to data files as well. It may not be as useful as policy files but if we're adding this anyways. We could introduce a new flag `--data` for example for providing policy and data files. WDYT?